### DISCUSSION TITLE
Release v1.6.2 / 5.61.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2542] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build introduces a couple of quality of life improvements, 
- the ability to suppress notification when gaming
- a global setting to add run entries to all boxes
- an option to select a custom icon for the sandbox added to the color slider in the box options
![grafik](https://user-images.githubusercontent.com/3890945/208420578-ac3dac40-b5d8-4eaa-bd99-66c9e05a5d73.png)

It fixed 12 issues and introduces a new update mechanism which allows to update individual files.

On the "Support Tab" in the "Global Options" the user can now choose from the following release channels:

![grafik](https://user-images.githubusercontent.com/3890945/208420986-d28373f1-8584-4455-ae39-83d2e9eec78d.png)

1. Stable - GitHub Releases
2. Preview - GitHub Pre-Releases

There the user can also select how to behave when a "New Version" (where an installer is available) or a "Version Update" (where only individual files of the existing installation will be updated) is found.
For a "New Version" the following options are available: Notify, Download & Notify, Download & Install
For a "Version Update" the following options are available: Ignore, Notify, Download & Notify, Download & Install
There is no "Ignore" option for "New Version" as that is covered by disabling check for updates.

In the "Stable" channel check for "Version Update" is only available to supporters with a valid certificate. In this channel all updates are signed and consists of the latest compatibility templates and urgent bug-fixes and translations.

In the "Preview" channel the "Version Update" consist of unsigned test builds (except the signed driver) released every few days (like for example 1.6.0,1.6.1a and 1.6.1b) as here the updates contain not only half tested fixes but also new functionality which may not yet be free of bugs.

Furthermore installers have been updated with the ability to download the latest Templates.ini and translations when installing.

### Added
- added template for Tencent TIM [#2516](https://github.com/sandboxie-plus/Sandboxie/pull/2516) (thanks TooYoungTooSimp)

### Added
- added game/presentation mode [#2534](https://github.com/sandboxie-plus/Sandboxie/issues/2534)
- added option to pick a custom box icon

### Fixed
- fixed support setting page not showing version updates properly

### Added
- added "Run Sandboxed" command to the sandbox menu
- DefaultBox can now be replaced with another sandbox [#2445](https://github.com/sandboxie-plus/Sandboxie/issues/2445)

### Fixed
- improved fix for [#2495](https://github.com/sandboxie-plus/Sandboxie/issues/2495)

### Added
- added global option to introduce run menu entries to all boxes

### Changed
- reorganized command prompt entries [#2451](https://github.com/sandboxie-plus/Sandboxie/issues/2451)
- the "live" update channel is now part of the "preview" channel for which version updates can now be disabled

### Fixed
- fixed issue with support setting drop-down menu multiplying [#2502](https://github.com/sandboxie-plus/Sandboxie/pull/2502) (okrc)
- added translations folder as deprecated, to be removed upon installation [#2500](https://github.com/sandboxie-plus/Sandboxie/pull/2500) (lufog)
- fixed issue with disk usage of hidden items not counted [#2503](https://github.com/sandboxie-plus/Sandboxie/pull/2503) (okrc)
- fixed issue with boolean settings which can also be set per process [#2495](https://github.com/sandboxie-plus/Sandboxie/issues/2495)
- fixed issue with disk usage of hidden items not counted [#2503](https://github.com/sandboxie-plus/Sandboxie/pull/2503)

### Added
- added ability to import/export boxes into 7z files
- added new update mechanism allowing for incremental updates
-- used for nightly test builds in the "live" update channel
-- used to update Templates.ini and translations after installation [#1105](https://github.com/sandboxie-plus/Sandboxie/issues/1105)

### Changed
- restructured the box context menu a bit
- SandMan translations are now compressed and located in translations.7z

### Fixed
- fixed issue with recycle icon not updated in time [#2457](https://github.com/sandboxie-plus/Sandboxie/issues/2457) (thanks okrc)
- fixed issue with "index out of range" [#2470](https://github.com/sandboxie-plus/Sandboxie/pull/2470) (thanks okrc)
- fixed issue starting SandMan after install [#2284](https://github.com/sandboxie-plus/Sandboxie/issues/2284)
- fixed encoding issue with my_version.h [#2475](https://github.com/sandboxie-plus/Sandboxie/issues/2475)
- fixed issue with empty group blinking sequence [#2486](https://github.com/sandboxie-plus/Sandboxie/pull/2486) (thanks okrc)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.2'>Release v1.6.2 / 5.61.2</a>.</em>

**Comment by marcinkit** [comment]
sandboxie classic 5.61.2 on win10;
SBIE2205 Service not implemented:  Wow64FunctionTurboThunkControl C00000BB

reverted to Sandboxie-Classic-x86-v5.59.2 which is the last release which is trouble free, error free.

**Reply by isaak654** [reply]
Just reported as #2546

**Reply by DavidXanatos** [reply]
This issue happens on 32 bit windows only, and can be safely ignored no need to switch back.

Sandboxie used for windows 7, 8 and 10 a hack to disable wow64 turbo thunking, on windows 10 there is however a documented way of disabling it hence in build 5.60.0 for windows 10 sandboxie switched to the documented method.
A chehck was missing to do nothing when the OS is 32 bit itself and no thunking is done at all.

**Comment by ghost** [comment]
If I understand correctly there is no way to set up SBIE to update individual files and do not notify about new full installers release. It would be nice to have it

**Reply by DavidXanatos** [reply]
mmh... without that notification how will you know that there is a new version?

**Comment by ghost** [comment]
I mean I don't need to be notified about new versions (installers) if I could just select an option to apply individual files updates

**Reply by DavidXanatos** [reply]
so why not set booth to auto download and install?

**Reply by ghost** [reply]
Installer overrides everything (even if files have not been changed) which means I cant have any sandboxed app running. Updating individual files _may_ not require closing any apps I assume

**Reply by DavidXanatos** [reply]
well except for templates.ini and translations.7z the file updater will only run when no boxes are active and will temporary stop the driver and service. So there is not really an advantage with regard to this.

**Reply by ghost** [reply]
Oh, ok, thanks for the reply

**Comment by howie9** [comment]
Install of 64bit version 5.61.2 failed. Using Windows 10 64bit
![can't install 64bit  5 61 2](https://user-images.githubusercontent.com/22160734/209037237-e20f8dec-f6ef-4a6f-aa72-b1af9dfd9b05.jpg)

**Reply by offhub** [reply]
1. Close the running sandboxed programs.
2. `Sandboxie Control` > `File` > `Exit`
3. From Windows Run Menu (WIN+R): Type `services.msc` and enter.
4. Find the `Sandboxie Service` > Right click on it > Select `Stop` (wait until its stopped)
5. Try to install again.

**Reply by howie9** [reply]
That worked, thank you

**Comment by howie9** [comment]
Version 5.61.4 64bit on Win 10 will not 
install.  I stopped the Sandboxie service, which 
worked before, but this time I keep getting this error:
Emacs!

  A reboot followed by another try to install the program was successful.
What can I do to identify the issue?

At 14:58 22-12-22, you wrote:

**Reply by isaak654** [reply]
I would suggest to open a [new report](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose) by providing all required details with a screenshot of the error.

**Comment by howie9** [comment]
Thanks, will do next time, if there is one.


At 03:17 02-01-23, you wrote:
